pkg/scaffold/operator.go

```diff
@@ -60,7 +60,7 @@ spec:
     spec:
       containers:
         - name: {{.ProjectName}}
-          image: REPLACE_IMAGE // replace this with the built image.
+          image: REPLACE_IMAGE
```

Comment:
I am not sure if the first time user knows what to do with the replace_image without going over the user guide. I'd suggest we have a comment here to tell user to replace this placeholder with the image path that you built.

Reply:
We have instructions on the user-guide/README docs to replace this placeholder or update the image after build:
```sh
sed -i 's|REPLACE_IMAGE|quay.io/example/app-operator|g' deploy/operator.yaml
```
So this was just to shorten the matched string.
I can add a yaml comment on top:
```YAML
# Replace this with the built image name
image: REPLACE_IMAGE
```

Reply:
@hasbro17 I understand. For example, If I am a user and I looked at the user-guide first time. I roughly know what's up. But let's see after a while, I don't recall anything concrete from the user guide. By providing comments in the the generated code, it can help me remember what I need to replace and also how to properly use the sdk. Essentially, I think that providing redundancy helps user to learn about operator-sdk project faster and better.